coralproject/talk-fe
# Replies Feature

## What does this PR do?

Allows users to reply to a comment in talk. Replies are optimistically rendered in the DOM so that they appear instantaneously.

## How do I test this PR?

1. From talk_fe directory `npm install && learn bootstrap`
2. `cd packages/coral-embed-stream`
3. `npm run dev-start`
4. When posted, unstyled comments should appear immediately.
5. Replies should be posted to the server in the appropriate format (check the browser dev panel.)
6. On reload, comments should be loaded from the server.

## Notes

1. Until backend work on views is complete relationships are being mocked in the getItemsQuery action. 
2. Tests are currently breaking for components with lerna-enabled imports. Until this is solved, tests are disabled for reply components. I am working to debug tests, but don’t want to hold up the PR.
